src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/InfluxDBHelper.java

```diff
@@ -341,7 +342,8 @@ public void ingestToInfluxDb(
             // TODO: TEMPORARILY COMMENTED BELOW TO INVESTIGATE "java.lang.OutOfMemoryError: Java heap space"
             // TODO: Right now, ingestion service is crashing with "Java heap space" issue and following
             // TODO: line shows up in the log as the reason.
-            //backupService.writeToBackup(payload, new URL(baseUrl), databaseName, retPolicyName);
+            // re-enable for testing
+            backupService.writeToBackup(payload, new URL(baseUrl), databaseName, retPolicyName);
```

Comment:
Perhaps introduce a config property to enable/disable and just default it accordingly.

Reply:
I like the idea of adding a flag for this feature. It will also help when we are deploying this in an env where we are using InfluxDB Enterprise. We might not want to use this feature in that env.